Dialogue/Afin/od_010200391.csv

```diff
@@ -1,8 +1,8 @@
-11_voice_NPC_5003_104#0,"""Somehow, you've got the courage to drink somewhat incomprehensible things ah, alone."""
-text02#0,"""If you are going to have a buddy, I'd be happy to drink it, yeah."""
-text03#0,"""Let's drink for a moment then.<br>It's seriously provided within the ARKS Ship and it's okay."""
-text04#0,"""Drinks made by the Medical Center is easy to fear until the end..."""
-text05#0,"""...Eeeee, man, that's heartbreaking.<br>Let's go, let's go. Another glass!"""
+11_voice_NPC_5003_104#0,"""Somehow, you've got the courage to drink somewhat incomprehensible things alone."""
+text02#0,"""If you're going to have a buddy, I'd be happy to drink it."""
+text03#0,"""Let's drink for a moment, then.<br>It's seriously provided within the ARKS ship, and it's okay."""
+text04#0,"""Drinks made by the Medical Center are easy to fear until the end..."""
+text05#0,"""...Eeeeh, man, that's heartbreaking.<br>Let's go, let's go. Another glass!"""
```

Comment:
Ok, noe buildbot's spelling checking do not take `Eeeeh`
you MAY want to update `_aspell/PSO2.en_Chat.dict`